controllers/dashboard.go

```diff
@@ -0,0 +1,37 @@
+package controllers
+
+import (
+	"github.com/beego/beego/v2/server/web"
+)
+
+// DashboardController operations for Dashboard
+type DashboardController struct {
+	BaseController
+}
+
+// NestPrepare prepares some configurations to the controller
+func (c *DashboardController) NestPrepare() {
+	c.actionPolicyMapping()
+}
+
+// URLMapping maps dashboard controller actions to functions
+func (c *DashboardController) URLMapping() {
+	c.Mapping("Index", c.Index)
+}
+
+// actionPolicyMapping maps dashboard controller actions to policies
+func (c *DashboardController) actionPolicyMapping() {
```

Comment:
AT first, I was not sure about the naming. However, now I can see how it's consistent with the previous `URLMapping` method 🌟